### PR TITLE
LPS-181688 Replace getCN with classNames

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://issues.liferay.com/browse/LPS-181688)

## Steps 

1. Go to Product Menu > People > Segments
2. Create a segment

### CODE REVIEW

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarCollapse.js [hunk 1]
```diff
@@ -101,18 +101,19 @@ const CriteriaSidebarCollapse = ({
...
-					`sidebar-collapse-${propertyGroup.propertyKey}`,
-					activeClasses
-				);
-
 				return (
-					<li className={sidebarCollapseListClasses} key={key}>
+					<li
+						className={classNames(
+							'sidebar-collapse-item',
+							`sidebar-collapse-${propertyGroup.propertyKey}`,
```
Comment (victorg1991) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarCollapse.js:112]: Same here, just merge those strings into one

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/BooleanInput.js [hunk 2]
```diff
@@ -24,14 +24,12 @@ function BooleanInput({className, disabled, onChange, propertyLabel, value}) {
...
 
-	const classes = getCN('criterion-input', 'form-control', className);
-
 	return (
 		<ClaySelectWithOption
 			aria-label={`${propertyLabel}: ${Liferay.Language.get(
 				'select-option'
 			)}`}
-			className={classes}
+			className={classNames('criterion-input', 'form-control', className)}
```
Comment (victorg1991) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/BooleanInput.js:32]: We can merge the two first string into one :)

**Review by victorg1991 — CHANGES_REQUESTED** [review]
Just minor details @DiegoHu97 :)

**Comment by DiegoHu97** [comment]
@victorg1991 updated! 😄

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
Many thanks @DiegoHu97  :DD

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by DiegoHu97** [comment]
Hi @victorg1991! I have just updated the snapshots 😄

**Review by victorg1991 — APPROVED** [review]

**Pull request closed without merging**